hbase-client/src/main/java/org/apache/hadoop/hbase/client/TableDescriptorBuilder.java

```diff
@@ -554,6 +554,11 @@ public TableDescriptorBuilder setRegionServerGroup(String group) {
     return this;
   }
 
+  public TableDescriptorBuilder unsetRegionServerGroup() {
```

Comment:
Maybe call it clearRegionServerGroup?

Reply:
Thanks for reviewing and comments, I've renamed it and added UTs.